cmake/valgrind-linux.supp

```diff
@@ -0,0 +1,18 @@
+{
+   <ubuntu-18.04-casadi-ipopt>
+   Memcheck:Leak
+   match-leak-kinds: definite
+   fun:malloc
```

Comment:
Just wondering, if this is set, won't prevent ``valgrind`` to find any memory leak?

Reply:
This fine has been generated by running Valgrind with the option `--gen-suppressions=all` In theory should affect only the memory leak reported in  https://github.com/dic-iit/bipedal-locomotion-framework/pull/61#issuecomment-656039676